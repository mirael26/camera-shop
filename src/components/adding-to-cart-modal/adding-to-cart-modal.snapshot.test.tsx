import { renderWithReduxAndRouter } from '../../test/helpers/render-with-redux-and-router';
import { productMock } from '../../test/mocks';
import AddingToCartModal from './adding-to-cart-modal';

jest.mock('./adding-to-cart-modal-card/adding-to-cart-modal-card', () => 'AddingToCartModalCard');
jest.mock('./adding-to-cart-modal-success/adding-to-cart-modal-success', () => 'AddingToCartModalSuccess');

test('AddingToCartModal renders correctly', () => {
  const { asFragment } = renderWithReduxAndRouter(<AddingToCartModal/>, { initialState: {
    cart: {
      addedToCartItem: productMock,
    }
  }});
  expect(asFragment()).toMatchSnapshot();
});
