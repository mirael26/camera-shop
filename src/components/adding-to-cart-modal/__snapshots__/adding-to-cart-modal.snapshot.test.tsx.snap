// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddingToCartModal renders correctly 1`] = `
<DocumentFragment>
  <div
    class="modal__content"
  >
    <addingtocartmodalcard />
  </div>
</DocumentFragment>
`;
